### PR TITLE
feat: Import 1.0.0 and tag 1.0

### PR DESCRIPTION
**Pull request by rhuss**

### CODE REVIEW

File: tools/images/migrate-images.sh [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
+  docker tag ${FROM_REGISTRY}/${image} ${TO_REGISTRY}/${image}
+  head_image=$(echo $image | sed -e 's/^\(.*\)\(\.[^.]*\)$/\1/')
+  docker tag ${TO_REGISTRY}/${image} ${TO_REGISTRY}/${head_image}
+  docker push ${TO_REGISTRY}/${image}
+  docker push ${TO_REGISTRY}/${head_image}
 done
+
+# Push s2i builder image
+S2I_IMAGE_SRC="registry.access.redhat.com/jboss-fuse-6/fis-java-openshift:2.0-9"
+S2I_IMAGE_TRG="fuse-ignite-java-openshift:$SYNDESIS_VERSION"
```
Comment (jimmidyson) [inline tools/images/migrate-images.sh]: typo: `TRG` -> `TAG`

Comment (jimmidyson) [reply tools/images/migrate-images.sh]: Oh you mean `TARGET` maybe? Sorry got confused!

**Review by jimmidyson — APPROVED** [review]

**Comment by jimmidyson** [comment]
Conflict...

File: tools/images/migrate-images.sh (hunk 1 above)
Comment (rhuss) [reply tools/images/migrate-images.sh]: Yeah, I'm excellent in creating silly acronyms ...

**Review by jimmidyson — APPROVED** [review]
LGTM. We should move to getopts or something to handle args better rather than just positional.

We also need to update templates to use 1.0 again...

**Pull request merged**